root.sh

```diff
@@ -39,6 +39,11 @@ incremental_recipe: |
 
   cmake --build . -- ${JOBS:+-j$JOBS} install
   mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+
+  echo "Building with external Alien plugin, removing interal plugin handlers."
+  rm -v "$INSTALLROOT/etc/plugins/TGrid/P010_TAlien.C"
```

Comment:
This will break the day they remove AliEn from ROOT. Changing it to `-vf` for forward compatibility. I am taking care of it.